config/parser.go

```diff
@@ -0,0 +1,79 @@
+package config
+
+import (
+    "io/ioutil"
+    "encoding/json"
+    "fmt"
+)
+
+type Config struct {
+    Db *Db
+    Email *Email
+    Sreality *Sreality
+}
+
+type Db struct {
+    Engine string `json:"engine"`
+    Host string `json:"host"`
+    Port int `json:"port"`
+    Username string `json:"username"`
+    Password string `json:"password"`
+    Database string `json:"database"`
+}
+
+type Email struct{
+    To string `json:"to"`
+    From string `json:"from"`
+    Server string `json:"server"`
+    Tls bool `json:"tls"`
+    TlsPort int `json:"tls_port"`
+    Username string `json:"username"`
+    Password string `json:"password"`
+}
+
+type Sreality struct{
+    Url string `json:"url"`
+    UrlDetail string `json:"url_detail"`
+    RealityType int `json:"reality_type"`
+    OperationType int `json:"operation_type"`
+    RealityOptions []int `json:"reality_options"`
+    Country int `json:"country"`
+    Region []int `json:"region"`
+    District []int `json:"district"`
+    PageResults int `json:"page_results"`
+    EstateAge int `json:"estate_age"`
+    Square *rangeInt `json:"square"`
+    Price *rangeInt `json:"price"`
+}
+
+type rangeInt struct {
+    Min int `json:"min"`
+    Max int `json:"max"`
+}
+
+func MustNewConfig(path *string) *Config {
```

Comment:
@grubastik why do you need a pointer on `path` variable? When I see `func SomeFunc(var *string)` - I expect that you are going to change `var` inside the function and after the call to `SomeFunc` I will have update value.
Change it to `MustNewConfig(path string) *Config {`